### PR TITLE
COMPASS-801 Add instance commands

### PR DESCRIPTION
**Pull request by pzrq**
This aims to consolidate all instance-model calls in [fetch.js](https://github.com/mongodb-js/instance-model/blob/master/lib/fetch.js) to here.

### CODE REVIEW

File: test/data-service-store.test.js [hunk 1]
```diff
@@ -33,6 +33,68 @@ describe('DataServiceStore', function() {
...
+      var unsubscribe = Actions.hostInfoComplete.listen(function(error, result) {
+        expect(error).to.equal(null);
+        expect(result.system).to.have.all.keys([
+          'currentTime', // 2017-02-22T07:01:35.999Z,
+          'hostname',    // 'rocket.local:27018',
+          'cpuAddrSize', // 64,
+          'memSizeMB',   // 16384,
+          'numCores',    // 4,
+          'cpuArch',     // 'x86_64',
+          'numaEnabled'  // false },
```
Comment (KeyboardTsundoku) [inline test/data-service-store.test.js]: random closing brace?

Comment (pzrq) [inline test/data-service-store.test.js]: It's in a comment and came in from the minimal copy/paste change.

Comment (pzrq) [inline test/data-service-store.test.js]: https://github.com/mongodb-js/data-service/commit/999713ab1b22700911974d96ff637b3fc26d0bae

**Comment by pzrq** [comment]
Thanks @KeyboardTsundoku @rueckstiess 👍

**Pull request merged**